test/test_habitat_task.py

```diff
@@ -24,6 +24,7 @@ def test_task_actions():
     config.freeze()
 
     env = habitat.Env(config=config)
+    assert env.action_space.contains(env.action_space.sample())
     env.reset()
     env.step(
         action={
```

Comment:
That maybe even better to move line 27 here like:
`env.action_space.contains(action)`